### ISSUE TITLE
[bug]: experimental feature "expand navigation" doesn't work

### ISSUE THREAD

**Issue by ANSI-C** [hoppscotch/hoppscotch#3938]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

when I open an API via link url the sidebar doesn't get automatically expanded to the API call I land on.

### Steps to reproduce

1. go to an api
2. click "share request" then select "link", then "create", then "copy link" (this procedure is rather long and clumsy BTW)
3. close the browser tab and paste the link on a new tab
4. I should land on the previous call **with the treeview on the sidebar expanded to the same call**

### Environment

Production

### Version

Cloud

**Comment by JoelJacobStephen** [comment]
Hi @ANSI-C. Sharing a request does not provide others with access to your request or collections. Instead, it generates a link that references your request, and this link can be shared with anyone and they can generate a copy of your request. Could you help us understand what use case you were trying to achieve so that we can understand this better?

**Comment by ANSI-C** [comment]
Yes, we would like a mean to share same requests in a teamwork to be able to use/update them with a fixed link, possibly shown in the browser url, maybe pointed by url fragment, if not available otherwise.

**Comment by thetronjohnson** [comment]
@ANSI-C is there a reason why creating a new workspace and inviting your team to the workspace won't work for you? Workspace are generally the goto option for collaboration?

**Comment by ANSI-C** [comment]
what if I want to document the call in an internal ticketing or KB system?

**Comment by anwarulislam** [comment]
@thetronjohnson, this would be a valuable feature for the workspace maybe? a unique link for each request that can be easily shared to that team member.
